### PR TITLE
ROI selector and FreeBayes sanitizer scripts to be added

### PR DESCRIPTION
**Pull request by szilvajuhos**
The FreeBayes part is an attempt to make FreeBayes usable by changing the output VCF and filtering.
The selectROI.py script is to get smaller interesting parts (Region Of Interest) from the huge result BAMs. 
Do not merge yet, just writing documentation for the latter.

### CODE REVIEW

File: scripts/selectROI.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+import subprocess
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Selecting Region Of Interest from BAM files')
+    parser.add_argument('-v', help='List of VCF files to process like -v tumour.vcf,normal.vcf',required=False, dest="vcfs")
+    parser.add_argument('-a', help='The BAM files (indexed) to read from like -b tumour.bam,normal.bam', required=True, dest="bams")
+    parser.add_argument('-b', help='The BED file with gene coordinates', required=False, dest="beds")
+    parser.add_argument('-p', help='The prefix of the target BAM file to write to (postfix is the input filename)', required=False, dest="prefix", default="ROI.")
+    parser.add_argument('-t', help='Number of threads used by samtools [8]', required=False, dest="threads", default=8)
+    parser.add_argument('-w', help='Region width for VCF loci [200]', required=False, dest="width", default=200)
```
Comment (marcelm) [inline scripts/selectROI.py]: If you add `type=int` here, you don’t need the `int(args.width)` below.

Comment (marcelm) [reply scripts/selectROI.py]: And `required=False` is the default, I’d omit it.

**Comment by szilvajuhos** [comment]
Thanks @marcelm , now it is fixed, ready for merge.

**Comment by marcelm** [comment]
I haven’t done a full review, just briefly looked at the Python script, so I don’t feel qualified to merge.

**Pull request merged**